packages/graphql-js-operation-payload-generator/src/index.ts

```diff
@@ -239,6 +240,12 @@ function visitDocumentDefinitionNode(
             ),
           };
           return fieldWithMockData;
+        } else if (namedType instanceof GraphQLEnumType) {
```

Comment:
Best to avoid references to graphql-js classes, as it tends to be the cause of runtime mismatches when dealing with [the dreaded] multiple versions of the same package. I fixed it here: https://github.com/microsoft/graphitation/pull/37